app/src/main/java/com/github/polypoly/app/ui/game/HeadsUpDisplay.kt

```diff
@@ -66,22 +68,19 @@ fun Hud(
     location: String
 ) {
     val playerState = gameViewModel.getPlayerStateData().observeAsState().value
-    val playerPosition =
-        mapViewModel.goingToLocationProperty?.name ?: "unknown destination" // TODO: use state data
+    val playerPosition = mapViewModel.goingToLocationProperty?.name ?: "unknown destination" // TODO: use state data
 
-    Column(modifier = Modifier.testTag("hud")) {
-        HudPlayer(playerData)
-        HudOtherPlayersAndGame(otherPlayersData, round)
-        HudLocation(location, testTag = "interactable_location_text")
-        if (playerState == PlayerState.MOVING)
-            HudLocation(
-                playerPosition,
-                DpOffset(0.dp, 80.dp),
-                "going_to_location_text",
-                "Going to: "
-            )
-        HudGameMenu()
-    }
+    HudLocation(location, testTag = "interactable_location_text")
+    if (playerState == PlayerState.MOVING)
+        HudLocation(
+            playerPosition,
+            DpOffset(0.dp, 80.dp),
+            "going_to_location_text",
```

Comment:
here I would specify the name of the parameters for readability, to know which string do what